site/en/install/lang_c.ipynb

```diff
@@ -148,7 +148,7 @@
         "  </tr>\n",
         "  <tr>\n",
         "    <td>Windows GPU only</td>\n",
```

Comment:
This could use a big warning that tensroflow no longer supports GPU on native windows. Like this one:

https://www.tensorflow.org/install/pip#windows-native

```
<aside class="caution">
  <b>Caution:</b> TensorFlow 2.10 was the last TensorFlow release that supported GPU on native-Windows. Starting with TensorFlow 2.11, you will need to install [TensorFlow in WSL2](https://tensorflow.org/install/pip#windows-wsl2), or install tensorflow-cpu and, optionally, try the [TensorFlow-DirectML-Plugin](https://github.com/microsoft/tensorflow-directml-plugin#tensorflow-directml-plugin-)
</aside>
```